akari/router.py

```
@@ -1,15 +1,26 @@
 
 
+@dataclasses.dataclass
+class _AkariRouterLoggerOptions:
+    info: bool = False
+    duration: bool = False
+
+
 class _AkariRouter:
-    def __init__(self, logger: logger._AkariLogger) -> None:
+    def __init__(
+        self, logger: logger._AkariLogger, options: _AkariRouterLoggerOptions = _AkariRouterLoggerOptions()
```

Ruff (0.8.2)

Do not perform function call `_AkariRouterLoggerOptions` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)